README.md

```diff
@@ -2,113 +2,174 @@
 
 ## Learning Goals
 
-- Find elements using a simple condition with `Array.prototype.indexOf()`
-- Find elements using more complex conditions with `Array.prototype.find()`
+- Find elements using `Array.prototype.find()`
+- Find elements using `Array.prototype.indexOf()`
 
 ## Introduction
 
-As developers, one of the things we need to do on a regular basis is locate
-things in arrays. It's all well and good to be able to store data, but it's
-pretty useless unless we're able to get it back out again. In JavaScript, there
-are two different methods that we use to locate data in arrays. For simple
-conditions, we use `Array.prototype.indexOf()`. For more complex calculations,
-we use `Array.prototype.find()`.
-
-## Find Elements Using a Simple Condition with `Array.prototype.indexOf()`
-
-`Array.prototype.indexOf()` is called on an array and takes two arguments: the
-value you are looking for and an optional start position. It compares each
-element in turn to the value you're looking for using the strict equality
-operator (`===`) and returns the index of the first matching element. If the
-element isn't contained in the array, it returns -1.
-
-```js
-
-const cards = ['queen of hearts', 'jack of clubs', 'ten of diamonds', 'ace of spades'];
-
-cards.indexOf('jack of clubs'); //=> 1
-cards.indexOf('jack of hearts'); //=> -1
-
-```
-
-If you pass in the optional second argument, `indexOf()` will begin the search
-at the specified position:
+| Use Case                                                       | Method                |
+| -------------------------------------------------------------- | --------------------- |
+| Executing a provided callback on each element                  | `forEach()`           |
+| ** Finding a single element that meets a condition             | `find()`, `indexOf()` |
+| Finding and returning a list of elements that meet a condition | `filter()`            |
+| Modifying each element and returning the modified array        | `map()`               |
+| Creating a summary or aggregation of values in an array        | `reduce()`            |
+
+In the last lesson, we learned about the `forEach()` iterator method. In this
+lesson we'll look at two iterator methods that help us locate data in arrays:
+`Array.prototype.find()`, which can be used for a wide array of search
+conditions, and `Array.prototype.indexOf()`, which can be used to search for a
+specific value.
+
+As you work through this lesson, you will notice that there is a lot of overlap
+in how `forEach()` and `find()` work (`indexOf()` is a bit different). This will
+be the case for all of the iterator methods you will learn about in this
+module other than `indexOf()`. Specifically, they all:
+
+1. iterate through the elements in the array they're called on,
```

Comment:
Noticed this in the `forEach` lesson as well, but maybe for consistency in lists make these unconnected sentences instead? i.e. 

1. Iterate through the elements in the array they're called on.
2. Take a callback as an argument.
3. Automatically pass value to the callback.
4. Return a value at the end (with the exception of `forEach()`).